src/Sulu/Component/Content/Tests/Unit/Document/Subscriber/FallbackLocalizationSubscriberTest.php

```diff
@@ -135,19 +135,6 @@ public function testNoWebspaceDoNotLoadGhostContent()
         $this->subscriber->handleHydrate($this->hydrateEvent->reveal());
     }
 
-    /**
-     * It should throw an exception if no locale can be determined.
-     *
-     * @expectedException \RuntimeException
-     */
-    public function testNoLocale()
```

Comment:
There seems to be no exception thrown in the `FallbackLocalizationSubscriber`.

The TestCase itself did only work because it did not implemented an expected call:

> $this->hydrateEvent->getOption('load_ghost_content', true)->willReturn(false);

and prophecy did call `UnexpectedCallException` which did extend from RuntimeException
so the test were green before.